libvast/src/system/configuration.cpp

```diff
@@ -51,22 +51,20 @@ void initialize_factories() {
 
 configuration::configuration() {
   detail::add_message_types(*this);
-  // Use 'vast.conf' instead of generic 'caf-application.ini'.
-  auto config_paths = std::vector<path>{};
+  // We must clear the config_file_path first so it does not use
+  // `caf-application.ini` as fallback. Instead we use our own `config_paths`
+  // variable to support multiple configuration files.
```

Comment:
This comment doesn't make sense, especially the "Instead..." part because the first sentence is not a negative one. Could you clarify?